packages/grafana-icons/package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "clean": "rimraf ./dist ./compiled ./package.tgz ./src/icons-gen",
     "generate": "yarn clean && npx @svgr/cli ./svg --silent && mv ./src/icons-gen/index.ts ./src",
-    "typecheck": "tsc --emitDeclarationOnly false --noEmit",
+    "typecheck": "yarn generate && tsc --emitDeclarationOnly false --noEmit",
```

Comment:
@Clarity-89 FYI - it seems Nx 19 now sees the icons package. I've had to add this generate in front of the typecheck to prevent CI failing.